graphql_api/types/bundle_analysis/base.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Mapping
```

Comment:
Added lots of Unimplemented resolvers here for coming PRs and also implemented a new way to present size and load times. Also marked the old ways of calculating it to be removed when it is fully migrated on the FE.